diff-hl-show-hunk-popup.el

```diff
@@ -1,10 +1,50 @@
 ;;; diff-hl-show-hunk-popup.el --- popup backend for diff-hl-show-hunk -*- lexical-binding: t -*-
 
+;; Copyright (C) 2020  Free Software Foundation, Inc.
+
+;; Author:   Dmitry Gutov <dgutov@yandex.ru>
+;;           Álvaro González <alvarogonzalezsotillo@gmail.com>
+;; URL:      https://github.com/dgutov/diff-hl
```

Comment:
The URL header is unnecessary here, it's only read from the main .el file by the package.el infrastructure.